ci/scripts/check_ci.sh

```diff
@@ -76,14 +78,14 @@ for pr in ${pr_list}; do
   if [[ "${num_cases}" -eq 0 ]] && [[ -d "${pr_dir}/RUNTESTS" ]]; then
     "${GH}" pr edit --repo "${REPO_URL}" "${pr}" --remove-label "CI-${MACHINE_ID^}-Running" --add-label "CI-${MACHINE_ID^}-Passed"
     "${GH}" pr comment "${pr}" --repo "${REPO_URL}" --body-file "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
-    sed -i "/${pr}/d" "${GFS_CI_ROOT}/${pr_list_file}"
+    "${HOMEgfs}/ci/scripts/pr_list_database.py" --remove_pr "${pr}" "${pr_list_dbfile}"
     # Completely remove the PR and its cloned repo on sucess of all cases
     rm -Rf "${pr_dir}" 
     continue 
   fi
 
   for cases in "${pr_dir}/RUNTESTS/"*; do
-    pslot=$(basename "${cases}")
+    pslot=$(basename "${cases}") || true
```

Comment:
Likely should be a non-zero exit.

Reply:
Yes, but should not be non-zero because it is an expected non-error condition.

Reply:
done